resources/views/inc/dashboard-head.blade.php

```diff
@@ -3,7 +3,7 @@
         @if(Carbon\Carbon::now()->month == 12)
             <a class="navbar-brand" href="/dashboard"><img width="100" src="/photos/xmas_logo.png"></a>
         @else
-            <a class="navbar-brand" href="/dashboard"><img width="100" src="/photos/logo.png"></a>
+            <a class="navbar-brand" href="/dashboard"><img width="100" src="/photos/ztl_logo_black-turkey.png"></a>
```

Comment:
Let's maybe add an else-if and check if it's month 11 so this will be automatic every year. Not a hard change and will keep us from having to change it back.